src/app/board/board.module.ts

```diff
@@ -3,10 +3,12 @@ import { CommonModule } from '@angular/common';
 
 import { BoardRoutingModule } from './board-routing.module';
 import { BoardComponent } from './board.component';
+import { HeaderModule } from '@app/header/header.module';
+import { SharedModule } from '@app/shared/shared.module';
+import { ListComponent } from './list/list.component';
 
 @NgModule({
-  declarations: [BoardComponent],
-  imports: [CommonModule, BoardRoutingModule],
+  declarations: [BoardComponent, ListComponent],
+  imports: [CommonModule, BoardRoutingModule, HeaderModule, SharedModule],
```

Comment:
Do we need a HeaderModule here?

Reply:
In the next pull request, I will separate the materials module and shared module and then put the header module in shared